File: tournaments/isSkewSymmetricMatrix/isSkewSymmetricMatrix.cs
bool isSkewSymmetricMatrix(int[][] matrix) {
    for (int i = 0; matrix.Length > i; ++i)
        for (int j = 0; matrix[0].Length > j; ++j)
            if (matrix[i][j] != -matrix[j][i])
                return false;
    return true;
}

File: tournaments/isSkewSymmetricMatrix/isSkewSymmetricMatrix.cpp
bool isSkewSymmetricMatrix(std::vector<std::vector<int>> matrix) {
  for (int i = 0; i < matrix.size(); i++) {
    for (int j = 0; j < matrix.size(); j++) {
      if (matrix[i][j] != -matrix[j][i]) {
        return false;
      }
    }
  }
  return true;
}
